### PR TITLE
Semistandardized README

### PR DESCRIPTION
**Pull request by RichardLitt**
## Description

This is an interesting project, because it isn't just a code package. As such, Standard README may not be the best for it. Still, I collapsed some sections, made it a bit easier to read. I think directing people to the readme.io docs makes a lot of sense - what you have here is basically extra information. Some of it is important to state twice, some of it less so.

Who is the code licensed under? That is not clear.

I hope this helps!

## Related Issue
https://github.com/orgmanager/orgmanager/pull/225

## Motivation and Context
See https://github.com/orgmanager/orgmanager/pull/225.

## How Has This Been Tested?
Used Markdown Preview in Atom to make sure it looks nice. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] None of the above. Doc fix.

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by m1guelpf** [comment]
@RichardLitt Thank you for your time!
Just a few notes I'd like to discuss:

- I think the Hosted Version should be moved to a separate category, as it can lead to confusion under the install category.
- I'd change the text in the install section, as it now seems like the instructions are the same, but different requirements are needed for the production version, and that isn't true.
- Should Testing and Updating be inside of Usage? I believe Updating should be in the same section as Installing, and maybe testing should be moved to a different category to avoid confusions.

**Comment by RichardLitt** [comment]
@m1guelpf Agreed. How does this look?

**Comment by m1guelpf** [comment]
@RichardLitt Looks awesome! What do you think it isn't clear about the license?

**Comment by RichardLitt** [comment]
Who holds the license?

**Comment by m1guelpf** [comment]
@RichardLitt _The OrgManager contributors?_ I didn't know I had to specify that? How can I do it?

**Comment by RichardLitt** [comment]
Pushed.

**Comment by m1guelpf** [comment]
@RichardLitt Awesome, thank you!

**Comment by m1guelpf** [comment]
@RichardLitt One last thing, should I add the OrgManager Logo at the top of the README?

**Comment by RichardLitt** [comment]
Yeah! Between the header and the badges :)

**Comment by m1guelpf** [comment]
@RichardLitt What I was thinking of is using the big logo and replacing with the OrgManager test:
logo

**Comment by RichardLitt** [comment]
> replacing with the OrgManager test:

What do you mean?

File: readme.md [hunk 1]
```diff
@@ -1,41 +1,53 @@
 # OrgManager
```
Comment (m1guelpf) [inline readme.md]: Replacing that with the image

Comment (m1guelpf) [inline readme.md]: Ok, thanks!

**Pull request merged**